### ISSUE TITLE
Main page design

### ISSUE THREAD

**Issue by jedgarbennett** [codeforkansascity/street-medicine-kc#4]
We need a create a front page describing what this page is about.  I have the pictures to use along with the language.

**Comment by tangospring** [comment]
Post the language in the comment here, Jay.

**Comment by jedgarbennett** [comment]
I leave this area to discussion on what it should look like.

**Comment by tangospring** [comment]
I understand. You have mentioned that you have the language for the main page. Can you post the text in the comment here?

Thanks!

**Comment by alfredosmkc** [comment]
Something to consider if the target users are agencies...images are less important than functionality. For example, if a case manager is delayed a few seconds every time she logs on to the site to find resources, it will increase frustration over time. I.E. Professionals within this service deal with this population daily and do not require pictures or images to make the UI appealing...clean design and ease of functionality will have more impact.

**Comment by tangospring** [comment]
I thought the target audience are homeless. Am I wrong?

**Comment by alfredosmkc** [comment]
According to Jae, the target users for the site are homeless agencies (which consists of agency staff, social workers, case workers, etc)...different needs for agency users.

**Comment by alfredosmkc** [comment]
From project goals: "Our goal is to provide an interactive tool for front line providers to the homeless population. This comprehensive digital resource guide allows easier access to agencies in seven counties. It will allow agencies to track their referrals."

**Comment by tangospring** [comment]
Oh, then it is an entirely different site. Several approaches I have considered are now irrelevant. Thanks.

Also personas would be different: not homeless, but agency staff.

**Comment by tangospring** [comment]
So this is incorrect, or at least misleading: "Provide Kansas City's homeless and organizations that provide services to them, the opportunity to connect with resources via better information about health and community services."

**Comment by alfredosmkc** [comment]
The above statement would probably need more clarification.

Well, whatever approaches you came up with previously are still valid. During the inception at the first hackathon, the homeless individual was the primary user focus. Since then, the user focus has pivoted to the agency user. However, it was the thought that the site would have interfaces for both types of users. So everything you've come up with is still relevant! :)

**Comment by tangospring** [comment]
Not really. Homeless person goals and abilities are substantially different from the goals of agency staff. For example, homeless needs shelter, food, open hours etc, Agency needs capacity, maybe a phone book.

**Comment by alfredosmkc** [comment]
The thought would be that the site would act as a landing portal to two
different interfaces.

On Sep 4, 2016 5:18 PM, "Oleh Kovalchuke" notifications@github.com wrote:

Not really. Homeless person goals and abilities are substantially different
from the goals of agency staff. For example, homeless needs shelter, food
etc, Agency needs capacity.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
https://github.com/codeforkansascity/street-medicine-kc/issues/4#issuecomment-244636990,
or mute the thread

**Comment by tangospring** [comment]
It is a rather ubiquitous trap in software development covered in any decent interaction design course. Here is an example: 

![car for multiple personas](https://wiki.fluidproject.org/download/attachments/3904542/Personas.gif)

**Comment by alfredosmkc** [comment]
Which is why the current focus is the agency user only. But a homeless user
site would still be in the pipeline.

**Comment by tangospring** [comment]
Yep, that is a different project with different goals and design. Lets not mix the two.
I assume the primary goal for this project is "a phone book of homeless agencies on the web". Is this approximation correct?

**Comment by alfredosmkc** [comment]
To simplify, yes.

**Comment by tangospring** [comment]
Companies pivot whenever they find a better source of revenue. 
What was the reason for pivot in this project?
